appengine-java8/pubsub/README.md

```diff
@@ -53,6 +53,21 @@ gcloud beta pubsub subscriptions create <your-subscription-name> \
   --ack-deadline 30
 ```
 
+- Create a subscription for authenticated pushes to send messages to a Google Cloud Project URL such as https://<your-project-id>.appspot.com/authenticated-push.
+
+The push auth service account must have Service Account Token Creator Role assigned, which can be done in the Cloud Console [IAM & admin](https://console.cloud.google.com/iam-admin/iam) UI.
+`--push-auth-token-audience` is optional. If set, remember to modify the audience field check in `PubSubAuthenticatedPush.java` (line 36).
```

Comment:
You can provide a link to the file if you wish.
`[PubSubAuthenticatedPush.java](..../PubSubAuthenticatedPush.java#L36)`